nestjs/src/courses/interviews/interviews.controller.ts

```diff
@@ -76,7 +76,7 @@ export class InterviewsController {
   @ApiForbiddenResponse()
   @ApiBadRequestResponse()
   @ApiOperation({ operationId: 'getAvailableStudents' })
-  @RequiredRoles([CourseRole.Mentor, CourseRole.Supervisor, CourseRole.Manager])
+  @RequiredRoles([Role.Admin, CourseRole.Mentor, CourseRole.Supervisor, CourseRole.Manager], true)
```

Comment:
why do you need admin? isn't it allowed by default?

Reply:
nope

Reply:
what a pity